src/plugins/intel_gna/src/gna_plugin.hpp

```diff
@@ -172,36 +176,37 @@ class GNAPlugin : public InferenceEngine::IInferencePlugin {
 
     void DumpXNNToFile() const;
 
-    void ImportFrames(void *ptr_dst,
-                     const void *ptr_src,
-                     InferenceEngine::Precision input_precision,
-                     float scaleFactor,
-                     intel_dnn_orientation_t orientation,
-                     uint32_t num_frames,
-                     uint32_t num_group,
-                     uint32_t num_vector_elements,
-                     uint32_t num_vector_stride);
-
-    void ExportScores(void *ptr_dst,
-                     const void *ptr_src,
-                     intel_dnn_orientation_t orientation,
-                     uint32_t num_frames,
-                     uint32_t num_group,
-                     uint32_t num_vector_elements,
-                     uint32_t num_active_elements,
-                     uint32_t num_vector_stride,
-                     InferenceEngine::Precision precision_in,
-                     InferenceEngine::Precision precision_out);
+    void ImportFrames(void* ptr_dst,
```

Comment:
I think ImportFrames, ExportsScores and CopyIputData can be moved to separate class or made as free Functions- due the fact they are not Tide to the GNAPlugin.
One of the mehtod would require extra argument or eventually GNAConfig.
What is more I would check if such number of argument is required.
Maybe it would be good to connect them to some structures/classs which would help to make argument list shorter.